tests/gold_tests/h2/httpbin.test.py

```diff
@@ -124,9 +124,10 @@
 test_run.Processes.Default.Streams.stderr = "gold/httpbin_3_stderr.gold"
 test_run.StillRunningAfter = httpbin
 
-
-# Check Logging
+# Wait for log file to appear, then wait one extra second to make sure TS is done writing it.
 test_run = Test.AddTestRun()
-test_run.DelayStart = 10
-test_run.Processes.Default.Command = 'echo "Delay for log flush"'
+test_run.Processes.Default.Command = (
+    os.path.join(Test.Variables.AtsTestToolsDir, 'condwait') + ' -f ' +
+    os.path.join(ts.Variables.LOGDIR, 'access.log') + ' ; sleep 1'
+)
```

Comment:
If I'm understanding your condwait shell script correctly, it incorporates the sleeps.  Why do you append an extra sleep at the invocation sites?

Reply:
For an extra safety margin, to ensure that TS has time to finish writing the log file before its contents are checked.

Reply:
Why don't you include that in the script then?

Reply:
OK done.